buddyboss/buddyboss-platform
# #1610 tags are lost upon updating discussion

### All Submissions:


### Changes proposed in this Pull Request:

Fixes #1610

### How to test the changes in this Pull Request:

1. Create a discussion with a few tags
2. Edit the discussion
3. See the tags in tags section

### Proof Screenshots or Video

https://www.loom.com/share/0e45a53eff144c4eaaecfcb24ba920e8

### Other information:


### Related Theme PR:
buddyboss/buddyboss-theme#1136
